view/frontend/web/js/navigation-form.js

```diff
@@ -183,10 +184,13 @@ define([
                 href = seoHref ? seoHref : href;
             }
 
-            let url = new URL(href, window.location.origin);
-            url.search = this._getFilterParameters();
+            if (this.options.urlStrategy === 'Tweakwise\\Magento2Tweakwise\\Model\\Catalog\\Layer\\Url\\Strategy\\QueryParameterStrategy') {
```

Comment:
Is there a better way than check on class path?

Reply:
I will move the class check to the navigation config. But there is no way to skip the class check entirety since this is the result of the getUrlStrategy check.